build/Jenkinsfile

```diff
@@ -0,0 +1,33 @@
+node() {
+    try {
```

Comment:
Hi, Please use the updated Jenkinsfile here as discussed in the call. Just copy and paste that Jenkinsfile from `samagra-devops` and change the `service name` to your `user-service`

Reply:
https://github.com/Samarth-HP/samarth-devops/blob/master/Jenkinsfile

Reply:
okay its done